hadoop-build-tools/src/main/resources/checkstyle/checkstyle.xml

```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0"?>
 <!DOCTYPE module PUBLIC
-    "-//Puppy Crawl//DTD Check Configuration 1.2//EN"
-    "http://www.puppycrawl.com/dtds/configuration_1_2.dtd">
+    "-//Checkstyle//DTD Checkstyle Configuration 1.2//EN"
+    "https://checkstyle.org/dtds/configuration_1_2.dtd">
```

Comment:
This is from HADOOP-16232. I've cherry picked that patch on its own, so the changes to the checkstyle DTDs can all be left out of this PR.

Reply:
Ok. I have rebased and removed this commit.